src/ImageSharp/Formats/Tiff/PhotometricInterpretation/BlackIsZero1TiffColor.cs

```diff
@@ -25,6 +26,8 @@ public override void Decode(ReadOnlySpan<byte> data, Buffer2D<TPixel> pixels, in
 
             int offset = 0;
 
+            var black = new Rgba32(0, 0, 0, 0xff);
```

Comment:
you can use named colors here instead `Color.White`  and `Color.Black`

Reply:
Oh, yeah, thanks 😌

Reply:
Done 798603f46cd92c21c4cf0aa74a2e2082b83d7ae7